src/features/place-bid/PlaceBidForm/FormSteps/hooks/useFormSteps.tsx

```diff
@@ -46,28 +47,34 @@ export const useFormSteps = ({
 	step,
 	error,
 	statusText,
-	tokenBalance,
 	bidAmount,
 	setBidAmount,
 	onCheckZAuction,
 	onApproveZAuction,
 	onConfirmPlaceBid,
 	onClose,
-}: useFormStepsProps): useFormStepsReturn => {
+}: useFormStepsProps): UseFormStepsReturn => {
+	const { paymentTokenSymbol, isLoadingTokenBalance } = usePlaceBidData(zna);
+
```

Comment:
Added timeout here to prevent jolt of fast loading token balance data. If token balance has loaded, the loader will not be displayed and the timeout will not fire.


https://user-images.githubusercontent.com/39112648/199768036-5a178359-3002-4e32-b8a8-117301e74f90.mov

Reply:
Nice - I like it